outlines/integrations/llamacpp.py

```diff
@@ -33,7 +33,7 @@
 from numpy.typing import NDArray
 from pydantic import BaseModel
 
-from outlines.fsm.guide import CFGGuide, Guide, RegexGuide
+from outlines.fsm.guide import CFGGuide, Guide, RegexGuide, StopAtEOSGuide
```

Comment:
The individual integrations are being removed, they're no longer used. https://github.com/outlines-dev/outlines/issues/1054